libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser/NBGeocoderMetadataParser.m

```diff
@@ -0,0 +1,74 @@
+//
+//  NBGeocoderMetadataParser.m
+//  libPhoneNumber-GeocodingParser
+//
+//  Created by Rastaar Haghi on 7/1/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "NBGeocoderMetadataParser.h"
+
+@implementation NBGeocoderMetadataParser {
+@private
+  SQLiteDatabaseConnection *_databaseConnection;
+}
+
+- (instancetype)init {
+  self = [super init];
+  return self;
+}
+
+- (void)convertFileToSQLiteDatabase:(NSString *)completeTextFilePath
+                       withFileName:(NSString *)textFileName
+                       withLanguage:(NSString *)languageCode {
+  NSString *fileName = completeTextFilePath;
+  NSData *data = [NSData dataWithContentsOfFile:fileName];
+  NSString *myStr = [[NSString alloc] initWithData:data
+                                          encoding:NSUTF8StringEncoding];
+  NSMutableArray<NSString *> *countryCodes =
+      (NSMutableArray *)[textFileName componentsSeparatedByString:@"."];
+  NSString *countryCode = countryCodes[0];
+  _databaseConnection =
+      [[SQLiteDatabaseConnection alloc] initWithCountryCode:countryCode
+                                               withLanguage:languageCode];
+  @autoreleasepool {
+    // This program splits each line of the text file into the two important
+    // pieces of info: phone number prefix and the corresponding region
+    // description.
+    NSCharacterSet *separator = [NSCharacterSet newlineCharacterSet];
+    NSArray *mArr = [myStr componentsSeparatedByCharactersInSet:separator];
+    NSMutableArray<NSString *> *keyValuePair = [[NSMutableArray alloc] init];
```

Comment:
Just declare this to be:
NSArray<NSString *> *keyValuePair is fine. No need to assign it.

Reply:
done.